CHANGELOG.rst

```diff
@@ -42,6 +42,18 @@ Changed
 Fixed
 ~~~~~
 
+* Fixed a bug where secrets in pack configs weren't being masked. Recently we
```

Comment:
Minor thing - we use 100 characters wide lines. Not a big deal and this can be changed in a subsequent PR.